src/services/dexes/uniswapV3/ActiveLiquiditySDK.ts

```diff
@@ -202,6 +202,10 @@ export class ActiveLiquititySDK {
         liquidityNet: tick.liquidityNet,
       },
     ];
+
+    [token0, token1] =
```

Comment:
maybe a method if we using this in two places?

Reply:
I created token utils. I will wait for confirmation before merge it